plugins/nuova/helpers.go

```diff
@@ -85,6 +85,9 @@ func checkRaidController(url, host string, client *retryablehttp.Client) (bool,
 			for retryCount < 1 && resp.StatusCode == http.StatusNotFound {
```

Comment:
Since err is not checked, and common.DoRequest returns either a nil err or nil resp, this probably means that there is an ull deref here in case of error. (the bug was already here AFAIU)

Reply:
Nevermind, there is only one retry, so this protects the resp.StatusCode. But then the loop in unnecessary!